src/promptflow/tests/sdk_cli_test/e2etests/test_cli.py

```diff
@@ -502,14 +518,18 @@ def test_pf_flow_test_with_additional_includes(self):
             node_name,
         )
 
+    @pytest.mark.skipif(
+        os.environ.get("PF_RECORDING_MODE", None) == "replay",
```

Comment:
too many skips, can we reduce the number of them? may skip in class?

Reply:
Let me skip them and create a pipeline.

a lot of skips because of the fetch_url is getting different content from different language.

I'll replace the web-classification in the future.